Header header

# Status
robot_actions/ActionStatus status

# Target position and orientation
ConstrainedGoal goal

# Actual position and orientation
robot_actions/Pose2D feedback